README.md

```
@@ -1 +1,120 @@
+
+```shell
+mvn install -DskipTests -Dpowsybl.docker.install
+```
+
+Please read [liquibase usage](https://github.com/powsybl/powsybl-parent/#liquibase-usage) for instructions to automatically generate changesets. After you generated a changeset do not forget to add it to git and in `src/main/resources/db/changelog/db.changelog-master.yml`.
+
+---
+
+## Interactions with Other Microservices
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)